byterun/sys.c

```diff
@@ -309,7 +309,9 @@ CAMLprim value caml_sys_getcwd(value unit)
 #ifdef HAS_GETCWD
   if (getcwd(buff, sizeof(buff)) == 0) caml_sys_error(NO_ARG);
 #else
-  if (getwd(buff) == 0) caml_sys_error(NO_ARG);
+  #ifdef HAS_GETWD
```

Comment:
`caml_sys_time_include_children_unboxed` indent sub-conditionals (even in `else if` position), `caml_sys_random_seem` does not. Pick your favorite choice :-)